.cursor/skills/dev-helper/lessons/communication.md

```
@@ -0,0 +1,18 @@
+# Lessons: Communication
+
+PR descriptions, Jira comments, review replies, commit messages, and
+human interaction patterns.
+
+## Entry Format
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)